pittsburgh/__init__.py

```diff
@@ -38,32 +38,107 @@ def get_organizations(self):
+
+        govt_services_committee = Organization(name="Committee on General and Government Services",
+                                               classification="committee")
+        govt_services_committee.add_source(pgh_legistar_url, note="web")
+        yield govt_services_committee
+
+        telecom_committee = Organization(name="Committee on General Services & Telecommunications",
+                                         classification="committee")
+        telecom_committee.add_source(pgh_legistar_url, note="web")
+        yield telecom_committee
+
+        arts_committee = Organization(name="Committee on General Services, Technology & the Arts",
+                                      classification="committee")
+        arts_committee.add_source(pgh_legistar_url, note="web")
+        yield arts_committee
+
+        housing_committee = Organization(name="Committee on Housing, Economic Development & Promotion",
+                                         classification="committee")
+        housing_committee.add_source(pgh_legistar_url, note="web")
+        yield housing_committee
+
+        parks_committee = Organization(name="Committee on Parks, Recreation & Youth Policy", classification="committee")
+        parks_committee.add_source(pgh_legistar_url, note="web")
+        yield parks_committee
+
+        zoning_committee = Organization(name="Committee on Planning, Zoning & Land Use", classification="committee")
+        zoning_committee.add_source(pgh_legistar_url, note="web")
+        yield zoning_committee
+
+        env_committee = Organization(name="Committee on Public Works & Environmental Services",
+                                     classification="committee")
+        env_committee.add_source(pgh_legistar_url, note="web")
+        yield env_committee
+
+        # for whatever reason these the clerk's office has also classified these last 3 as committees in Legistar
+        mayor_agenda = Organization(name="Mayor's Agenda - Legislation to be Presented", classification="committee")
+        mayor_agenda.add_source(pgh_legistar_url, note="web")
+        yield mayor_agenda
+
+        post_agenda = Organization(name="Post Agenda Meeting", classification="committee")
+        post_agenda.add_source(pgh_legistar_url, note="web")
+        yield post_agenda
+
+        hearing_sched = Organization(name="PUBLIC HEARING SCHEDULE", classification="committee")
+        hearing_sched.add_source(pgh_legistar_url, note="web")
+        yield hearing_sched
+
         mayor = Organization(name="Mayor", classification="executive")
         mayor.add_post("Mayor", "Mayor", division_id="ocd-division/country:us/state:pa/place:pittsburgh")
         mayor.add_source("http://pittsburghpa.gov/mayor/index.html", note="web")
         yield mayor
 
-        # TODO: figure out disambiguation for councilman/mayor positions (using birth_date?)
+        # TODO: figure out disambiguation for councilman/mayor positions—using birth_date isn't working
```

Comment:
@fgregg and I [talked about](https://github.com/opencivicdata/scrapers-us-municipal/pull/278#discussion_r297815760) how to distinguish between the mayoral and council terms of Peduto and Ravenstahl when I made my last PR, but unfortunately the suggestion of adding `birth_date` didn't work—got the same error when running locally with this section uncommented:

```
pupa.exceptions.SameNameError: multiple people with same name "William Peduto" in Jurisdiction - must provide birth_date to disambiguate
```

Reply:
did you add birth_date in both places you create a person?

Reply:
looks like the fix was actually to `add_term` for the same user ids rather than instantiate two different `Person`s